services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -164,6 +174,8 @@ approveDevice
     -> Galley Response
 approveDevice (zusr ::: tid ::: uid ::: connId ::: req ::: _) = do
     assertLegalHoldEnabled tid
+    Log.debug $ Log.field "target" (toByteString uid)
```

Comment:
For consistency sake I'd simply use `target` here too (even though it's just one)

Reply:
you probably mean `targets`?  plural?

Reply:
> you probably mean targets? plural?

yes